File: RafaelSanabria_GenerarTiempos.py
import time

def fibonacci(N):
    #Retorna el N esimo numero de Fibonacci
    if(N== 0 or N==1):
        return N
    else:
        return fibonacci(N-2)+fibonacci(N-1)

def get_time(N):
    #Retorna el tiempo que se demora en calcular el N esimo numero de fibonacci
    t0 = time.time()
    fibonacci(N)
    t1 = time.time() - t0
    return t1

#N y el tiempo que se demora en calcular fibonacci(N)
for i in range(36):
    print str(i) + "," + str(get_time(i))

File: RafaelSanabria_GenerarTiempos.cpp
#include <iostream>
#include <ctime>

//Calcula el N esimo numero de fibonacci
int fibonacci(int N){
  if(N==0 || N==1){
    return N;
  }
  else{
    return fibonacci(N-2)+fibonacci(N-1);
  }
}

//Calcula el tiempo que se demora en calcular fibonacci(N)
float get_time(int N){
  clock_t t;
  t = clock();
  fibonacci(N);
  t = clock() -t;
  float time = ((float)t)/CLOCKS_PER_SEC;
  return time;
}

//Imprime N y el tiempo en calcular fibonacci(N)
int main(){
  for(int i = 0;i<40;i++){
    float tiempo = get_time(i);
    std::cout<<i << "," << tiempo<<std::endl;
  }
}
